bin/kano-uixinit-systemd

```diff
@@ -21,61 +22,10 @@
 
 logger --id --tag "info" "uixinit systemd starts"
 
-# FIXME: is this still needed?
-mkdir -p "$HOME/.kano-settings"
-
-# Track some information about the hardware used
-kano-tracker-ctl generate hw-info
-
-# Disable XServer screen saver time and screen blanking (The display would become black)
-if [ -n "`which xset`" ]; then
-    xset s off
-    xset -dpms
-    xset s noblank
-fi
-
-# Calibrates the mouse pointer speed
-/usr/bin/startmouse
-
-# Set the initial default volume
```

Comment:
Don't we still need this when entering desktop mode? The reason this is here is because lxpanel sets the default, so we need to reset it again afterwards.